Sources/Logging/Log+FileLogDestination.swift

```diff
@@ -74,10 +72,11 @@ public extension Log {
                     let fileHandle = try FileHandle(forWritingTo: self.fileURL)
 
                     fileHandle.seekToEndOfFile()
+                    //swiftlint:disable force_unwrapping
```

Comment:
Change this to `//swiftlint:disable:next force_unwrapping`, so that we don't have to re-enable it afterwards 😛

Reply:
Didn't knew this one. Thank you. Will use it

Reply:
I had already mentioned it on the previous comment 😆 
You also have the `:this` variant, which is used inline